### DISCUSSION TITLE
Is there anyone who would like to follow each other on GitHub?

### DISCUSSION THREAD

**Discussion by theconomicat** [community/community#135530] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I want to make developer friends and increase my followers lol 

![a2e36b3ef43e422127b69f706bcdfd79572ad38f_high](https://github.com/user-attachments/assets/5a8caa89-eac2-44c9-88ad-53490b370bc6)

**Comment by ebndev** [comment]
Hi @love1ace, we’re glad you’re here 😀

Unfortunately, we currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
